deployment/terraform/variables.tf

```diff
@@ -402,6 +402,10 @@ variable "check_api_limits_schedule_expression" {
   default = "rate(1 hour)"
 }
 
+variable "clean_facilitylistitems_schedule_expression" {
```

Comment:
I tent to avoid running this command automatically this release. Let's run it manually.

Reply:
Agree.